docs/content/en/docs/getting-started/production-environment/snow-getstarted.md

```diff
@@ -15,8 +15,9 @@ See [Cluster topologies]({{< relref "../../concepts/cluster-topologies" >}}) for
 
 EKS Anywhere on Snow needs:
 
-* EKS Anywhere enabled Snowball devices. See [Ordering EKS anywhere enabled Snow Devices](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) for ordering experience through the AWS Snow Family console.
-* To be run on an Admin instance in a Snowball Edge device. See [Initial Device Setup](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) for setting up the devices, launching the Admin instance, fetching and copying the device credentials to the Admin instance for `eksctl` CLI to consume.
+* Certain pre-steps to complete before interacting with a Snowball device. See [Actions to complete before ordering a Snowball Edge device for Amazon EKS Anywhere](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html).
```

Comment:
We will replace these "whatissnowball.html" link with more specific links, once we know what they are, right?

Reply:
yup. currently all the snow doc reference links are all using `whatissnowball.html` to pass the resubmit checks. I will replace them once snow doc is alive